pkg/packing/packer.go

```diff
@@ -78,7 +78,7 @@ func (p *packer) Pack(ctx context.Context, constraints *Constraints, instances [
 			continue
 		}
 		packings = append(packings, packing)
-		zap.S().Debugf("Computed packing for pod(s) %v with instance type option(s) %s", apiobject.PodNamespacedNames(packing.Pods), instanceTypeNames(packing.InstanceTypeOptions))
+		zap.S().Infof("Computed packing for %d pod(s) with instance type option(s) %s", len(packing.Pods), instanceTypeNames(packing.InstanceTypeOptions))
```

Comment:
In my experience, this tends to run very verbose. Do these changes reduce how much is being printed here? If not, do you have an example, because imo I think this would be a lot to print in logs.

Reply:
I'm now printing just the number of pods, rather than all of the pods. Since we limit instance-types to 20, it significantly simplifies this.